### DISCUSSION TITLE
No cameras available

### DISCUSSION THREAD

**Discussion by doubleu88** [NestMTX/app#42] [General]
![image](https://github.com/user-attachments/assets/583a612e-32d6-4e19-b39c-f34c2b2864f4)

I have set up my credentials, but I still do not see my cameras listed. Am I missing something? I have the Nest doorbell camera and the Nest Hub max

**Comment by xthejuicex** [comment]
You need to click the pencil next to your credentials. This will open a Google page where you specify what cameras you want access granted to.
